### ISSUE TITLE
Deprecations Symfony 3.4

### ISSUE THREAD

**Issue by Sydney-o9** [KnpLabs/KnpGaufretteBundle#179]
Hi Team,

I noticed 1 deprecation in Symfony Profiler.

> User Deprecated: The "knp_gaufrette.filesystem_map" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public, or stop using the container directly and use dependency injection instead.

```
{▼
  /..../vendor/symfony/dependency-injection/Container.php:275 {▶}
  /..../vendor/knplabs/knp-gaufrette-bundle/KnpGaufretteBundle.php:35 {▼
    › if (empty($fileSystems)) {
    ›     $fileSystems = $this->container->get('knp_gaufrette.filesystem_map');
    ›     foreach ($fileSystems as $domain => $fileSystem) {
  }
  /..../vendor/symfony/http-kernel/Kernel.php:141 {▶}
  /..../vendor/symfony/http-kernel/Kernel.php:197 {▶}
  /../public/index.php:56 {▼
    › $request = Request::createFromGlobals();
    › $response = $kernel->handle($request);
    › $response->send();
  }
}
```

**Comment by nicolasmure** [comment]
Hello :)

Thank you for reportig this issue, this has been fixed in #176 and has been released in `v0.5`.
Can you try to upgrade the bundle please ?

**Comment by Sydney-o9** [comment]
Great! That fixed it. Thank you @nicolasmure
